server/server_test.go

```diff
@@ -451,74 +451,6 @@ func TestNewMetrics(t *testing.T) {
 	}
 }
 
-func TestRegisterRetryMiddleware(t *testing.T) {
```

Comment:
Regarding the test I left a comment in the second paragraph of my comment above https://github.com/containous/traefik/pull/1793#issuecomment-316662439. Please take a look.